.gitignore

```diff
@@ -1,4 +1,4 @@
-zdns/zdns
+zdns
```

Comment:
This will ignore the zdns folder

Reply:
currently we don't have a `zdns` folder at the root, but, if we want to have one, we'll need to rename the binary in order to keep ignoring it. Do you have a preference on which we do?